lib/Table/TableAutomaticStylesStratum.ts

```diff
@@ -136,13 +137,6 @@ export default class TableAutomaticStylesStratum extends LoadableStratum(
       );
     }
 
-    // If still no styles - try to create a style using text columns
-    if (columns.length === 0) {
-      columns = this.catalogItem.tableColumns.filter(
-        column => column.type === TableColumnType.text
-      );
-    }
-
     return columns.map((column, i) =>
```

Comment:
We don't need this now, as `TableMixin.defaultStyle` will handle it https://github.com/TerriaJS/terriajs/pull/5940/files#r756501263